.github/workflows/release.yml

```diff
@@ -66,6 +66,11 @@ jobs:
         with:
           path: ./CHANGELOGRELEASE.md
 
+      - name: Delete old drafts
+        uses: hugo19941994/delete-draft-releases@v1.0.0
```

Comment:
I like it!

However I have one minor concern: when the creation of a new release draft fails, we don't have any draft left over. We'd then have to do a new commit to trigger the action. If there's a simple way to not delete the last draft, I'd take this.

But not having one last draft is probably fine, too.

Reply:
At the moment, there is only an option to not delete the last drafts based on time passed (eg. keep the last n days of drafts).

Initially I had the same concern as you. But when I thought about it, I came to the conclusion that this might be a theoretical problem. We are deleting the drafts only if the Changelog is successfully generated. Once we are this far almost nothing can fail anymore. I can not remember an instance, where the creation of the release draft failed after the Changelog was created successfully.

Thinking about it ... we could put the deletion last (after the creation of the new release draft) and delete all drafts older than 10 minutes. That way we would always have at least one draft. This could run into problems, if the Action workflow runs very slowly or if there are time issues on the runner. How does that sound to you?

Reply:
I think you're right that there's practically no way the last action could fail. So I'm fine with this.

Deleting thew draft older than 10mins (or something along these lines) is probably risikier.